neurom/core/_neuron.py

```diff
@@ -291,15 +407,65 @@ def __str__(self):
     __repr__ = __str__
 
 
-class Neuron:
+class Neuron(morphio.mut.Morphology):
     """Class representing a simple neuron."""
 
-    def __init__(self, soma=None, neurites=None, sections=None, name='Neuron'):
-        """Initialize a Neuron object."""
-        self.soma = soma
-        self.name = name
-        self.neurites = neurites
-        self.sections = sections
+    def __init__(self, filename, name=None):
+        """Neuron constructor.
+
+        Args:
+            filename (str|Path): a filename
+            name (str): a option neuron name
+        """
+        try:
+            morphio.set_ignored_warning([morphio.Warning.appending_empty_section,
```

Comment:
stupid question: 
is this valid only for the `_neuron.py` module ?

Reply:
No, it's general. that's why I set it back in `finally`. However I didn't check this in parallel execution.